backend/fleet_management/serializers.py

```diff
@@ -164,3 +164,21 @@ def is_valid(self, raise_exception=False):
             ):
                 err.status_code = status.HTTP_409_CONFLICT
             raise err
+
+
+class RefuelSerializer(serializers.ModelSerializer):
+    driver = UserSerializer(read_only=True)
+    car = CarSerializer()
```

Comment:
```suggestion
```

and add that in Meta - https://www.django-rest-framework.org/api-guide/serializers/#specifying-read-only-fields